.ci.yaml

```diff
@@ -84,6 +84,29 @@ targets:
       - cipd_packages/codesign/**
       - .ci.yaml
 
+  - name: Mac ruby
+    bringup: true
+    recipe: cocoon/cipd
+    properties:
+      script: cipd_packages/ruby/build.sh
+      cipd_name: flutter/ruby/mac-amd64
+      device_type: iOS
```

Comment:
Is the device_type needed?

Reply:
Yes, we want these scripts to run on devicelab machines as they depend on brew being installed.